lib/elixir/lib/kernel.ex

```diff
@@ -6093,6 +6093,143 @@ defmodule Kernel do
     Macro.compile_apply(mod, fun, [code, options, __CALLER__ | args], __CALLER__)
   end
 
+  hour_in_ms = 1000 * 60 * 60
+  day_in_ms = 24 * hour_in_ms
+  week_in_ms = 7 * day_in_ms
```

Comment:
I might have missed something, but rather than needing to unquote these, we might simply be able to use attributes?
```suggestion
  @hour_in_ms 1000 * 60 * 60
  @day_in_ms 24 * @hour_in_ms
  @week_in_ms = 7 * @day_in_ms
```

Reply:
Yeah we can but I always go with `unquote` to save it storing the attr or whatever. I have no preference.

Reply:
You can’t use module attributes in Kernel anyway.

Reply:
Oh man there ya go. Call it instinct then 😄

Reply:
But I think bootstrap is failing anyway. :) You are probably still using stuff that is not yet available in Kernel.

Reply:
Yep sorry had to step away, looking